### PR TITLE
Add Webb.ai integration tile

### PR DESCRIPTION
**Pull request by anirudhwebbai**
### What does this PR do?

Added tile for "Webb.ai" API-based integration.

### Motivation

- Become a market partner
- Integrate our services with Datadog to troubleshoot monitors

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors.

### CODE REVIEW

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:5]:
```suggestion
Matt by Webb.ai is the first AI-enabled reliability engineer. It automates troubleshooting to identify the root cause in less than 5 mins. Matt troubleshoots alerts from Observability tools like Datadog, cloud providers like AWS, Azure, and Google Cloud, and infrastructure like Kubernetes.
```

Also, the root cause of what? An incident? Please add that.

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:9]:
```suggestion
Visit [Webb.ai][2] to sign up for free. Once you register, you can start the integration from the Webb.ai tile on the Datadog integrations page.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:13]:
```suggestion
To remove the Datadog integration from Webb.ai, navigate to the [Webb.ai integrations page][1] and delete the Datadog integration from the list.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:15]:
```suggestion
Once this integration has been uninstalled, any previous authorizations are revoked. 

To ensure that all API keys associated with this integration are disabled, search for the integration name on the [Datadog API Keys][4] page.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:35]:
```suggestion
[4]: https://app.datadoghq.com/organization-settings/api-keys

```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:20]:
```suggestion
Webb.ai does not generate any metrics; it relies on Datadog metrics.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:5]:
```suggestion
Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots alerts from observability tools like Datadog, cloud providers like AWS, Azure, and Google Cloud, and incidents from infrastructure like Kubernetes. It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.
```

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:5]: The overview should include: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What specific data your integration monitors and the value?

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:10]: This section should list out the exact steps the user has to take to install the integration. For example, 1. Click Install on the tile, 2. Click connect accounts 3. (oauth steps).... etc

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:28]: Please update this to reflect what data you are sending in

File: webb_ai/assets/dashboards/webbai_rca.json
Comment (emarsha94) [inline webb_ai/assets/dashboards/webbai_rca.json:2]: This should not link out to your own dashboard. This should be out of the box dashboard for users to look and visualize data

Comment (emarsha94) [reply webb_ai/assets/dashboards/webbai_rca.json:2]: Please use the template I sent over previously

File: webb_ai/manifest.json
Comment (emarsha94) [inline webb_ai/manifest.json:18]: Please update this to reflect the submitted data type as seen [here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags).

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Left a few comments. Additionally, please include your logo in svg format, at least 3 images showing the integration (graphs, ootb dashboard, etc) that will display on your integration tile, and add the codeowners file.

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:12]: Please include what data is specifically sent to datadog/what data is being monitored in Datadog and the value

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Looks great! Thank you for making the suggested changes :) Minor changes left!

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:18]: This is a great start but it would be helpful to provide the value in 1-2 sentences of the data being monitored. EX: 

This integration sends events to Datadog including RCAs performed by Matt and changes in kubernetes clusters <insert value of this information here> 

Ex of value: providing insight into xyz

Comment (emarsha94) [reply webb_ai/README.md:18]: We want to inform the user as to why this information is helpful to them

**Comment by HadhemiDD** [comment]
the images are causing the asset validation to fail, please fix it:
The aspect ratio for the image at `images/webb_ai-datadog-dashboard.png` is 1.96, but should be 16:9 (1.78).
The aspect ratio for the image at `images/webb_ai-rca-summary.png` is 2.31, but should be 16:9 (1.78).
The aspect ratio for the image at `images/webb_ai-troubleshooting-steps.png` is 2.83, but should be 16:9 (1.78).

**Review by HadhemiDD — DISMISSED** [review]
the images are causing the asset validation to fail, please fix it:
The aspect ratio for the image at images/webb_ai-datadog-dashboard.png is 1.96, but should be 16:9 (1.78).
The aspect ratio for the image at images/webb_ai-rca-summary.png is 2.31, but should be 16:9 (1.78).
The aspect ratio for the image at images/webb_ai-troubleshooting-steps.png is 2.83, but should be 16:9 (1.78).

File: webb_ai/README.md
Comment (sunstick) [reply webb_ai/README.md]: @emarsha94 Thanks for the feedback. We updated the text to reflect why we think this is useful to datadog users. Please take a look.

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:11]:
```suggestion
This integration queries the following data:
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:15]:
```suggestion
This integration sends events to Datadog, including the root cause analysis performed by Matt and changes in Kubernetes clusters.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:16]:
```suggestion
You can view detailed RCA analysis for your alerts, including all hypotheses analyzed and the exact steps Matt took with supporting evidence.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:17]:
```suggestion
With this analysis, 80-90% of troubleshooting is automated, greatly reducing the debugging time of on-call engineers. 
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:21]:
```suggestion
1. Visit [Webb.ai][2] and sign up for the free service.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:22]:
```suggestion
2. Navigate to Webb.ai tile in the [Datadog integrations page][5] and click **Install Integration**.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:23]:
```suggestion
3. Go to the **Configure** tab and click **Connect Accounts**.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:24]:
```suggestion
4. Follow the series of OAuth steps to finish setting up the integration.
```

File: webb_ai/README.md
Comment (neko-dd) [inline webb_ai/README.md:43]:
```suggestion
- Root cause analyses performed by Matt
```

File: webb_ai/assets/dashboards/webb_ai_overview.json
Comment (neko-dd) [inline webb_ai/assets/dashboards/webb_ai_overview.json:39]:
```suggestion
              "content": "- Datadog monitors in alert state are continuously pulled by Matt\n- Once webb.ai receives the alert, it automatically troubleshoots it using relevant Datadog metrics and Kubernetes information\n- Once the troubleshooting is done, you can view the analysis and complete steps that Matt has taken\n- An event is sent to Datadog for every RCA done by Matt\n- An event is sent to Datadog for every Kubernetes change observed in your environment",
```

**Review by neko-dd — APPROVED** [review]
LGTM from the docs perspective unless any more prose changes occur - if so please ping for re-review

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:13]:
```suggestion
This integration pulls data from Datadog include events, metrics, and tags. Matt utilizes the queried data to troubleshoot and debug alerts decreasing MTTR. 
```

File: webb_ai/README.md
Comment (emarsha94) [inline webb_ai/README.md:17]:
```suggestion
With this integration, events are sent to Datadog including the root cause analysis performed by Matt and changes in Kubernetes clusters. The events include detailed RCA analysis for your alerts such as the steps Matt took and all analyzed hypotheses to reduce debugging time of on-call engineers. 
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]

File: webb_ai/README.md
Comment (neko-dd) [reply webb_ai/README.md:17]: This edit changes the meaning of the prose and I don't think it's correct.

**Review by emarsha94 — APPROVED** [review]

**Comment by emarsha94** [comment]
@HadhemiDD Could you do a final review on the PR? Thanks!

**Comment by emarsha94** [comment]
@neko-dd could you do a final review? Thanks!

**Review by steveny91 — APPROVED** [review]

**Review by james-eichelbaum — APPROVED** [review]

**Pull request merged**
